library/core/src/num/nonzero.rs

```diff
@@ -2265,3 +2265,69 @@ mod verify {
     nonzero_check!(u128, core::num::NonZeroU128, nonzero_check_new_unchecked_for_u128);
     nonzero_check!(usize, core::num::NonZeroUsize, nonzero_check_new_unchecked_for_usize);
 }
+
+#[cfg(kani)]
+mod macro_nonzero_check_max {
```

Comment:
Nice! Last thing -- instead of having a module for each function, can you move all of these harnesses inside of `mod verify`? We're trying to keep a consistent format for these proofs across challenges, and that's how we've been doing it so far.

Reply:
I have done this in NonZero (rotate_left & rotate_right) Proof #202

Reply:
Sure, but could you please also do it here?